### PR TITLE
fix: Randomize SSM parameter name for Grafana token

### PR DESCRIPTION
**Pull request by bonclay7**
### What does this PR do?

Since we create a hardcoded `/terraform-accelerator/grafana-api-key` parameter per usage of the `eks-monitoring` module for storing Grafana API key, this conflicts when using the module multiple times in the same region.

This creates a random suffix for when the key changes. 

### More

- [x] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [x] Yes, I ran `pre-commit run -a` with this PR
- [x] Yes, I have added a new example under [examples](https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/main/examples) to support my PR (when applicable)
- [x] Yes, I have updated the [Pages](https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/main/docs) for this feature

**Note**: Not all the PRs required examples and docs.

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: examples/existing-cluster-with-base-and-infra/cleanup.sh [hunk 1]
```diff
@@ -28,4 +28,4 @@ if [[ $? -eq 0 && $destroy_output == *"Destroy complete!"* ]]; then
 else
   echo "FAILED: Terraform destroy of all targets failed"
   exit 1
-fi
\ No newline at end of file
+fi
```
Comment (elamaran11) [inline examples/existing-cluster-with-base-and-infra/cleanup.sh:31]: What changed?

**Review by elamaran11 — COMMENTED** [review]
LGTM, minor comment.

File: examples/existing-cluster-with-base-and-infra/install.sh [hunk 2]
```diff
@@ -29,4 +29,4 @@ if [[ ${PIPESTATUS[0]} -eq 0 && $apply_output == *"Apply complete"* ]]; then
 else
   echo "FAILED: Terraform apply of all modules failed"
   exit 1
-fi
\ No newline at end of file
+fi
```
[the remark this answers is not in the document]
Comment (bonclay7) [reply examples/existing-cluster-with-base-and-infra/install.sh:32]: same as above

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**